Calculator/Calculator/Sources/Controller/ViewController.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  Calculator - ViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+final class ViewController: UIViewController {
+    
+    @IBOutlet private weak var currentOperatorLabel: UILabel!
+    @IBOutlet private weak var currentNumberLabel: UILabel!
+    
+    @IBOutlet private weak var verticalStackView: UIStackView!
+    
+    @IBOutlet private weak var scrollView: UIScrollView!
+    
+    private var stringToBeCalculated: String = NameSpace.emptyString
+    private var currentNumber: String = NameSpace.emptyString
+    private var isCalculated: Bool = false
+    
+    //MARK: viewDidLoad
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        initializeCurrentOperator()
+        verticalStackView.subviews.forEach { subView in
+            subView.removeFromSuperview()
+        }
+    }
+    
+    private func initializeCurrentOperator() {
+        currentOperatorLabel.text = NameSpace.emptyString
+    }
+    
+    //MARK: Methods inserting numbers to be calculated
+    @IBAction private func tapNumberButton(_ sender: UIButton) {
+        if isCalculated {
+            allClear()
+            isCalculated = false
+        }
+        guard currentNumber.replacingOccurrences(of: ",", with: NameSpace.emptyString).count < 20 else {
+            return
+        }
+        insertString(titleName: sender.titleLabel?.text)
+        setCurrentNumber(titleName: sender.titleLabel?.text)
+    }
+    
+    @IBAction private func tapDotButton(_ sender: UIButton) {
+        if currentNumber.contains(".") == false {
+            if currentNumber == NameSpace.emptyString {
+                currentNumber += NameSpace.stringZero
+                stringToBeCalculated += NameSpace.stringZero
+            }
+            insertString(titleName: sender.titleLabel?.text)
+            setCurrentNumber(titleName: sender.titleLabel?.text)
+        }
+    }
```

Comment:
아래와 같이 guard 문을 사용하시면 들여쓰기 depth를 줄일 수 있을 것 같아요.

```suggestion
    @IBAction private func tapDotButton(_ sender: UIButton) {
        guard currentNumber.contains(".") == false else {
            return
        }

        if currentNumber == NameSpace.emptyString {
            currentNumber += NameSpace.stringZero
            stringToBeCalculated += NameSpace.stringZero
        }

        insertString(titleName: sender.titleLabel?.text)
        setCurrentNumber(titleName: sender.titleLabel?.text)
    }
```

Reply:
guard문으로 수정하였습니다!